### PR TITLE
Feature branch - search list

### PR DESCRIPTION
**Pull request by bwell-bruce**
This branch will be merged at a future date. This PR will be used for comparing in the meantime.

### CODE REVIEW

File: src/utils/searchForm.util.js [hunk 1]
```diff
@@ -135,13 +151,87 @@ const getLastUpdate = function (req, modifier) {
     return dateString;
 };
 
+const formatDate = (res) => {
+    const dateString = res.meta && res.meta.lastUpdated ? res.meta.lastUpdated : '';
+    if (dateString === '') {
```
Comment (DarylCheung) [inline src/utils/searchForm.util.js:156]: Do you want to check if 'dateString' contains 'T' here too? If there isn't one then the code might error downstream

Comment (bwell-bruce) [reply src/utils/searchForm.util.js:159]: Reworked the function completely

File: src/utils/searchForm.util.js [hunk 2]
```diff
@@ -135,13 +151,87 @@ const getLastUpdate = function (req, modifier) {
...
+};
+
+const getTotalMessage = (res) => {
+    const pageIndex = getCurrentPageIndex(res.body._getpagesoffset);
+    const lowCount = searchLimit * pageIndex + 1;
+    const increaseCount = res.resources.length < searchLimit ? res.resources.length : searchLimit;
+    const highCount = searchLimit * pageIndex + increaseCount;
+    const paginationMessage = `${lowCount} to ${highCount} of found results`;
+    return isValidResource(res.resources[0], res.resourceDefinition.name) &&
+        res.resources.length > 0
```
[the remark this answers is not in the document]
Comment (bwell-bruce) [reply src/utils/searchForm.util.js]: Short circuit updated

File: src/views/partials/footer.ejs [hunk 3]
```diff
@@ -1,18 +1,26 @@
...
-    <div class="small text-end pt-2">
+    <div class="px-3 pt-2"><%= searchUtils.totalMessage(locals, total) %></div>
+    <% } %>
+    <div class="flex-grow-1 small text-end pt-2">
         &copy; Copyright <%= currentYear %> b.well Connected Health
     </div>
 </footer>
 
 <script type="text/javascript" src="/js/footernav.js"></script>
 <script type="text/javascript" src="/js/bootstrap.min.js"></script>
```
[the remark this answers is not in the document]
Comment (bwell-bruce) [reply src/views/partials/footer.ejs]: Good catch! Removed.

File: src/utils/searchForm.util.js (hunk 1 above)
Comment (bwell-bruce) [reply src/utils/searchForm.util.js:159]: Reworked the formatDate function

**Review by DarylCheung — APPROVED** [review]

**Pull request merged**